infra/kubernetes/main.tf

```
@@ -0,0 +1,88 @@
+// terraform config //
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+  # backend "s3" {
+  #   bucket  = "cncf-credits-infra-kubernetes-tfstate"
+  #   key     = "terraform.tfstate"
+  #   region  = "ap-southeast-2"
+  #   encrypt = true
+  # }
+}
+
+provider "aws" {
+  profile = "default"
+  region  = "ap-southeast-2"
+}
+// ---------------- //
+
+// the tfstate for the management of the kubernetes account+OU //
+resource "aws_kms_key" "cncf-credits-infra-kubernetes-tfstate" {
+  description = "This key is used to encrypt bucket objects"
+}
+resource "aws_s3_bucket" "cncf-credits-infra-kubernetes-tfstate" {
+  bucket = "cncf-credits-infra-kubernetes-tfstate"
+  acl    = "private"
+
+  lifecycle {
+    prevent_destroy = true
+  }
+
+  versioning {
+    enabled = true
+  }
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        kms_master_key_id = aws_kms_key.cncf-credits-infra-kubernetes-tfstate.arn
+        sse_algorithm     = "aws:kms"
+      }
+    }
+  }
+}
+resource "aws_s3_bucket_ownership_controls" "cncf-credits-infra-kubernetes-tfstate" {
+  bucket = aws_s3_bucket.cncf-credits-infra-kubernetes-tfstate.id
+
+  rule {
+    object_ownership = "BucketOwnerPreferred"
+  }
+}
+resource "aws_s3_bucket_public_access_block" "cncf-credits-infra-kubernetes-tfstate" {
+  bucket = aws_s3_bucket.cncf-credits-infra-kubernetes-tfstate.id
+
+  block_public_acls   = true
+  block_public_policy = true
+}
+// ----------------------------------------------------------- //
+
+resource "aws_organizations_organization" "Root" {
```

```suggestion
resource "aws_organizations_organization" "root" {
```
and concommitant changes.

---

The org is actually with the capital in the organizations view. Is this value here just symbolic or does it need to match exactly? - it can be changed if only is symbolic